source/errors.rst

```diff
@@ -155,7 +155,7 @@ username is provided.
 
   {
     "status": 401,
-    "error": "Invalid username",
+    "error": "Invalid Username",
```

Comment:
https://github.com/osuosl/timesync-node/blob/develop/src/errors.js#L138 shows `'Invalid username'` as the correct form.